packages/builder/src/builders/plugin.ts

```diff
@@ -13,9 +13,9 @@ function escapeStringRegexp(str: string) {
   return str.replace(/[|\\{}()[\]^$+*?.]/g, "\\$&").replace(/-/g, "\\x2d");
 }
 
-function getClientEnvironment(prefix: RegExp) {
+function getClientEnvironment(prefix: RegExp, envPath: string = ".env") {
```

Comment:
by adding a default value to ngxEnv builder options (previous comment), the default value here is not needed